artifacts/pom.xml

```diff
@@ -146,241 +160,253 @@
-					<classifier>jar-with-dependencies</classifier>
-					<type>jar</type>
-				</artifactItem>
-<!-- artifactItem section for childauthfilter jar -->
-				<artifactItem>
-					<groupId>io.mosip.authentication</groupId>
-					<artifactId>authentication-childauthfilter-impl</artifactId>
-					<version>${authentication-childauthfilter-impl.version}</version>
-					 <outputDirectory>${authentication-childauthfilter-impl.location}</outputDirectory> 
-					<destFileName>${authentication-childauthfilter-impl.fileName}</destFileName>
-					<type>jar</type>
-				</artifactItem>
-<!-- artifactItem section for hazelcast-all jar -->
-				<artifactItem>
-					<groupId>com.hazelcast</groupId>
-					<artifactId>hazelcast-all</artifactId>
-					<version>${hazelcast.version}</version>
-					 <outputDirectory>${hazelcast.location}</outputDirectory> 
-					<destFileName>${hazelcast.fileName}</destFileName>
-					<type>jar</type>
-				</artifactItem>
-<!-- artifactItem section for idp authentication wrapper - only required for idp v090-->
-				 <artifactItem>
-					<groupId>io.mosip.idp</groupId>
-					<artifactId>authentication-wrapper</artifactId>
-					<version>${authentication-wrapper.version}</version>
-					 <outputDirectory>${authentication-wrapper.location}</outputDirectory> 
-					<destFileName>${authentication-wrapper.fileName}</destFileName>
-					<type>jar</type>
-				</artifactItem>
-<!-- artifactItem section for esignet authentication wrapper -->
-				<artifactItem>
-					<groupId>io.mosip.esignet.mock</groupId>
-					<artifactId>mock-esignet-integration-impl</artifactId>
-					<version>${esignet-mock-wrapper.version}</version>
-					<outputDirectory>${esignet-wrapper.location}</outputDirectory>
-					<destFileName>${esignet-mock-wrapper.fileName}</destFileName>
-					<type>jar</type>
-				</artifactItem>
-					<artifactItem>
-						<groupId>io.mosip.authentication</groupId>
-						<artifactId>esignet-integration-impl</artifactId>
-						<version>${esignet-ida-wrapper.version}</version>
-						<outputDirectory>${esignet-wrapper.location}</outputDirectory>
-						<destFileName>${esignet-ida-wrapper.fileName}</destFileName>
-						<type>jar</type>
-					</artifactItem>
+						<!-- artifactItem section for redis-cache jar -->
+						<artifactItem>
+							<groupId>io.mosip.cacheprovider</groupId>
+							<artifactId>cache-provider-redis</artifactId>
+							<version>${redis-cache.version}</version>
+							<outputDirectory>${redis-cache.location}</outputDirectory>
+							<destFileName>${redis-cache.fileName}</destFileName>
+							<type>jar</type>
+						</artifactItem>
+						<!-- artifactItem section for idp authentication wrapper - only required for idp v090-->
+						<artifactItem>
+							<groupId>io.mosip.idp</groupId>
+							<artifactId>authentication-wrapper</artifactId>
+							<version>${authentication-wrapper.version}</version>
+							<outputDirectory>${authentication-wrapper.location}</outputDirectory>
```

Comment:
do we need to point to older version 0.9.0 of authentication-wrapper.
Also is this still needed by esignet . To be checked and decided as it is still pointing to very old release from 2023.

Reply:
Removed the authentication-wrapper and idp related folder post discussion.